### DISCUSSION TITLE
4.2.4

### DISCUSSION THREAD

**Discussion by nalexandru** [nalexandru/BlenderCompat#103] [Announcements]
Blender 4.2.4, commit [e41b0e807a59](https://projects.blender.org/blender/blender/commit/e41b0e807a59).
Build environment:

    Visual Studio 2019, version 16.11.42
    Windows SDK 10.0.22000
    NVIDIA CUDA 11.8
    NVIDIA OptiX 7.3

AMD HIP and SM3.0 kernel binaries copied from the official release.

<hr /><em>This discussion was created from the release <a href='https://github.com/nalexandru/BlenderCompat/releases/tag/4.2.4'>4.2.4</a>.</em>

**Comment by XDmoyang** [comment]
My god！ may I ask if there will be an update to version 4.4
